python/PyQt6/core/auto_generated/pointcloud/qgspointclouddataprovider.sip.in

```diff
@@ -170,80 +170,6 @@ Only providers which report the CreateRenderer capability will return a 2D rende
 providers will return ``None``.
 %End
 
-    virtual bool hasStatisticsMetadata() const;
```

Comment:
This is all an API break -- we need to keep all methods exposed to Python for the lifetime of 3.x

Reply:
@nyalldawson `QgsPointCloudDataProvider` has been intentionally marked in docstring as experimental, allowing changes at in future - so I believe we should allow API breaks here...